runbot_travis2docker/models/runbot_build.py

```diff
@@ -61,7 +58,8 @@ class RunbotBuild(models.Model):
     def get_docker_image(self, cr, uid, build, context=None):
         git_obj = GitRun(build.repo_id.name, '')
         image_name = git_obj.owner + '-' + git_obj.repo + ':' + \
-            build.name[:7] + '_' + os.path.basename(build.dockerfile_path)
+            build.name[:7] + '_' + os.path.basename(build.dockerfile_path) + \
```

Comment:
I'm agree with migrate this piece to string interpolation and other similar pieces of code.
But this is unrelated with the focus of this PR.

What about create a issue to work on other PR?